label_studio/core/settings/base.py

```diff
@@ -556,3 +556,46 @@ def collect_versions_dummy(**kwargs):
 APP_WEBSERVER = get_env('APP_WEBSERVER', 'django')
 
 BATCH_JOB_RETRY_TIMEOUT = int(get_env('BATCH_JOB_RETRY_TIMEOUT', 60))
+
+if get_env('MINIO_STORAGE_ENDPOINT') and not get_bool_env('MINIO_SKIP', False):
+    CLOUD_FILE_STORAGE_ENABLED = True
+    DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
+    AWS_STORAGE_BUCKET_NAME = get_env('MINIO_STORAGE_BUCKET_NAME')
+    AWS_ACCESS_KEY_ID = get_env('MINIO_STORAGE_ACCESS_KEY')
+    AWS_SECRET_ACCESS_KEY = get_env('MINIO_STORAGE_SECRET_KEY')
+    AWS_S3_ENDPOINT_URL = get_env('MINIO_STORAGE_ENDPOINT')
+    AWS_QUERYSTRING_AUTH = False
+    # make domain for FileUpload.file
+    AWS_S3_SECURE_URLS = False
+    AWS_S3_URL_PROTOCOL = 'http:' if HOSTNAME.startswith('http://') else 'https:'
+    AWS_S3_CUSTOM_DOMAIN = HOSTNAME.replace('http://', '').replace('https://', '') + '/data'
+
+if get_env('STORAGE_TYPE') == "s3":
+    CLOUD_FILE_STORAGE_ENABLED = True
+    DEFAULT_FILE_STORAGE = 'htx.storage.CustomS3Boto3Storage'
```

Comment:
`htx.storage.CustomS3Boto3Storage` does not exist in this repository

Reply:
thanks, I am moving it there